app_dart/lib/src/request_handlers/scheduler/request_subscription.dart

```diff
@@ -46,16 +46,22 @@ class SchedulerRequestSubscription extends SubscriptionHandler {
   @override
   Future<Body> post() async {
     BatchRequest request;
+    final String data = message.data!;
+    Map<String, dynamic> jsonData;
+    log.info('rawJson: $data');
     try {
-      final String rawJson = String.fromCharCodes(base64Decode(message.data!));
-      log.info('rawJson: $rawJson');
-      final Map<String, dynamic> json = jsonDecode(rawJson) as Map<String, dynamic>;
-      request = BatchRequest.fromJson(json);
-    } catch (e) {
-      log.severe('Failed to construct BatchRequest from message');
-      log.severe(e);
-      throw BadRequestException(e.toString());
-    }
+          jsonData = jsonDecode(data) as Map<String, dynamic>;
+        } on FormatException {
+          jsonData = json.decode(String.fromCharCodes(base64.decode(data))) as Map<String, dynamic>;
+        }
+
+    try {
+          request = BatchRequest.fromJson(jsonData);
+        } catch (e) {
+          log.severe('Failed to construct BatchRequest from message');
+          log.severe(e);
+          throw BadRequestException(e.toString());
+        }
```

Comment:
I suspect a format is needed here to pass the CQ.

Reply:
done